docs/low-code-sql/development/gems/datasources.md

```diff
@@ -0,0 +1,20 @@
+---
+title: Data Sources
+id: datasources-sql
+description: access your data
+sidebar_position: 1
+tags:
+  - SQL
+  - sources
+  - seeds
+---
+
+Constitutes the set of Gems that help with loading data.
+
+## Source
+
```

Comment:
How should we discuss the "Add a source" button? Will this allow browsing tables /views in the future?
<img width="1122" alt="image" src="https://github.com/SimpleDataLabsInc/prophecy-docs/assets/20563764/4ccc5e5f-15ef-4f75-9376-000222a9dbbb">

Reply:
If you want to you can add a source, but the preferred method is to drag and drop a table from the Env tab so you don't have to specify the table details manually.

Reply:
done